WordPress/WordPressTest/PostHelperJetpackSocialTests.swift

```diff
@@ -2,14 +2,10 @@ import XCTest
 
 @testable import WordPress
 
-class PostService_JetpackSocialTests: CoreDataTestCase {
+class PostHelperJetpackSocialTests: CoreDataTestCase {
```

Comment:
Just curious. Why did you decide to drop the `+` in the name of the file and the `_` in the name of het class?

Reply:
The functions that are tested by this test case are all in `PostHelper`, hence the renaming. As for the `_`, I just thought it looked a bit strange in a type name. 😅

Reply:
Ahah. Fair enough, a `_` _does_ look out of place in a type name.

I don't know if it's common or not, but I've seen it in other places used to map a test name with a `+`. Based on the previous name, here I would have expected the file to be called `PostHelper+JetpackSocialTests.swift` and the class `PostHelper_JetpackSocialTests`. But I do see how it's a bit of a custom thing that might not be intuitive if one hasn't seen it already.

I think it might be worth encoding the convention with a SwiftLint rule at some point—after one has been agreed upon, of course.

Pragmatically, the name of the test file and class doesn't matter. XCTest will run the tests alphabetically by default, and that's about it.